do_cmake.sh

```diff
@@ -18,7 +18,7 @@ if [ -r /etc/os-release ]; then
       fedora)
           PYBUILD="3.7"
           if [ "$VERSION_ID" -ge "32" ] ; then
-              PYBUILD="3.8"
+              PYBUILD="3"
```

Comment:
what if some developer installs python3.9 on fedora31/fedora32? on which python3.9 is not fully supported, and lots of packages are missing.